deploy/common/k8sbased.sh

```diff
@@ -293,14 +300,14 @@ function launch_central {
       export ROX_ADMIN_PASSWORD="$(< "${unzip_dir}/password")"
     fi
 
-    echo "Deploying Central..."
+    echo "Deploying Central to namespace ${namespace}..."
 
-    ${KUBE_COMMAND:-kubectl} get namespace "${STACKROX_NAMESPACE}" &>/dev/null || \
-      ${KUBE_COMMAND:-kubectl} create namespace "${STACKROX_NAMESPACE}"
+    ${KUBE_COMMAND:-kubectl} get namespace "${namespace}" &>/dev/null || \
+      ${KUBE_COMMAND:-kubectl} create namespace "${namespace}"
 
     if [[ -f "$unzip_dir/values-public.yaml" ]]; then
       if [[ -n "${REGISTRY_USERNAME}" ]]; then
-        $unzip_dir/scripts/setup.sh
+        ROX_NAMESPACE="${namespace}" "${unzip_dir}/scripts/setup.sh"
```

Comment:
qq: `ROX_NAMESPACE` is already supported by the setup script?

Reply:
The setup script expects the namespace in the variable `ROX_NAMESPACE`, yes. But this is not the variable the rest of the deploy scripts use. Compare with `deploy/common/env.sh`. I agree that this is unfortunate, but I didn't want to change the setup script, since that also ends up in customer visible deployment bundles.